content/index.md

```diff
@@ -4,29 +4,11 @@ subtitle: component-based templating
 description: expressive • declarative • composable • technology-agnostic
 safe: false
 
-Modern JavaScript frameworks come with batteries included and everything
-but the kitchen sink.
+complate is a JSX-based HTML templating library, providing component markup abstractions that are reusable accross multiple languages and frameworks.
 
-We do **_a lot less_** and we think that is a good thing.
-
-complate provides a lightweight [JSX](what-is-jsx.html)
-implementation to allow us to write declarative component-based templates
-(macros).
-
-Our goal is to provide a templating language that is optimized for
-server-side rendering and still allows us to get the benefits of
-[abstraction and composition](rationale.html) that come from using a
-component-based approach. complate works across many platforms, allowing
-us to get true reuse out of our components.
-
-complate differs from
-[web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components)
-in that the macros are designed primarily for use in server-side rendering.
-However, complate and web components actually make a great team, allowing us to
-develop both our server-side code and our client-side code using components.
 
 <div class="code-example">
-<div class="input" aria-label="Input">
+<div class="input" aria-label="You use your template like this...">
```

Comment:
This might be an FNDism, but I much prefer the more concise labels there. In my local WIP version, I'm now using "Input: JSX" and Output: HTML". 🤷‍♂

Reply:
I personally agree with you, but I actually changed this based on the feedback I got from Sonja.

If you are _not_ familiar with JSX and HTML and you come on the page and see this code example (which is one of the first things you see), it could be extremely confusing. I think _in this case_ it makes a lot of sense to have a longer label here in order to explain what is going on for people who are coming from a different background. So on the left-hand side I think we need a label like "This is what using a complate template looks like" and on the right-hand side "This is what the generated HTML looks like". It's ok if we find a more concise way to put that, but I feel like particularly for the left-hand side, we need to explicitly say "Using complate template" (that it's JSX is here an implementation detail).

Having more detailed labels will not harm a user who is already familiar with JSX, but I think it will really help someone who has never seen it before.